### DISCUSSION TITLE
[Lists]: Scrolling abnormality

### DISCUSSION THREAD

**Discussion by kjxbyz** [community/community#122788] [Mobile]
### Select Topic Area

Bug

### Body

https://github.com/community/community/assets/47768002/bee9af71-28c2-40f8-9280-06ae3bd44e91

**Comment by Iron-Ham** [comment]
Hi @kjxbyz,

Thank you so much for reporting this bug. I've been able to confirm and reproduce it using your list. I'll look at this immediately and should hopefully have this fixed soon.

User reports like this help us ensure that GitHub Mobile is working well for all users – so please report any other issue you may be experiencing! 

Thanks,
Hesham

**Reply by kjxbyz** [reply]
You're welcome, I'm a heavy GitHub user, and it's a good thing for me that the GitHub app is easy to use.

**Reply by Iron-Ham** [reply]
I'm happy to report that I have a fix in flight, which should optimistically be a part of next week's beta release (with a release to production the following week). 

Thank you again!

**Reply by kjxbyz** [reply]
@Iron-Ham This bug was not fixed in version 1.162.0.

**Reply by Iron-Ham** [reply]
@kjxbyz we rolled out 1.163.0 to users on our TestFlight beta – which includes the bug fix – earlier today. Around this time next week, it should be generally available in the App Store.
